server/src/main/java/io/druid/client/coordinator/CoordinatorClient.java

```diff
@@ -61,38 +62,44 @@ public CoordinatorClient(
     this.selector = selector;
   }
 
+  public List<String> findDatasources(List<String> dataSources)
```

Comment:
let's delete findDatasources() and the memcached import since this is no longer used